### PR TITLE
[ODS-5872] Swagger Add year and tenant dropdowns to swagger UI

### PR DESCRIPTION
**Pull request by mjaksn**

### CODE REVIEW

File: Application/EdFi.Ods.Features/Controllers/OpenApiMetadataController.cs [hunk 1]
```diff
@@ -32,11 +31,11 @@ namespace EdFi.Ods.Features.Controllers
     [AllowAnonymous]
     [ApplyOdsRouteRootTemplate]
     [Route("metadata")]
+    [Route("{tenantId}/metadata")]
```
Comment (simpat-jesus) [inline Application/EdFi.Ods.Features/Controllers/OpenApiMetadataController.cs:34]: this attribute is not needed; that route template is covered by OdsRouteRootTemplateConvention

File: Application/EdFi.Ods.Features/OpenApiMetadata/Providers/EnabledOpenApiMetadataDocumentProvider.cs [hunk 2]
```diff
@@ -26,22 +28,27 @@ public class EnabledOpenApiMetadataDocumentProvider : IOpenApiMetadataDocumentPr
...
         {
             _openApiMetadataCacheProvider = openApiMetadataCacheProvider;
             _routeInformations = routeInformations;
             this._reverseProxySettings = apiSettings.GetReverseProxySettings();
-
+            _multitenancyEnabled = apiSettings.IsFeatureEnabled(ApiFeature.MultiTenancy.GetConfigKeyName());
             _schemaNameMaps = new Lazy<IReadOnlyList<SchemaNameMap>>(schemaNameMapProvider.GetSchemaNameMaps);
-
+            _routeRootTemplateProvider = routeRootTemplateProvider;
+            _odsContextRoutePath = apiSettings.GetOdsContextRoutePath() ?? "";
```
Comment (simpat-jesus) [inline Application/EdFi.Ods.Features/OpenApiMetadata/Providers/EnabledOpenApiMetadataDocumentProvider.cs:51]: replace "" with string.Empty

**Review by simpat-jesus — CHANGES_REQUESTED** [review]
Looks good, some minor changes and can be merged

File: Application/EdFi.Ods.Features/OpenApiMetadata/Providers/EnabledOpenApiMetadataDocumentProvider.cs (hunk 2 above)
Comment (mjaksn) [reply Application/EdFi.Ods.Features/OpenApiMetadata/Providers/EnabledOpenApiMetadataDocumentProvider.cs]: Ok, I pushed the changes

**Pull request merged**